### PR TITLE
fix-relative-units - set text icon font size scoped on span (#974)

### PR DESCRIPTION
**Pull request by Kuzcoo**
@limonte : let me know if you think it's a viable solution.

### CODE REVIEW

File: src/sweetalert2.scss [hunk 1]
```diff
@@ -613,13 +613,16 @@ body {
   margin: $swal2-icon-margin;
   border: .25rem solid transparent;
```
Comment (limonte) [inline src/sweetalert2.scss]: let's replace `.25rem` with `.25em` to make the icon border relative to the modal base font-size.

File: src/utils/dom.js [hunk 2]
```diff
@@ -92,9 +92,15 @@ const sweetHTML = `
      <div class="${swalClasses.icon} ${iconTypes.error}">
        <span class="swal2-x-mark"><span class="swal2-x-mark-line-left"></span><span class="swal2-x-mark-line-right"></span></span>
      </div>
-     <div class="${swalClasses.icon} ${iconTypes.question}">?</div>
-     <div class="${swalClasses.icon} ${iconTypes.warning}">!</div>
-     <div class="${swalClasses.icon} ${iconTypes.info}">i</div>
+     <div class="${swalClasses.icon} ${iconTypes.question}">
+       <span class="swal2-icon-text">?</span>
```
Comment (limonte) [inline src/utils/dom.js]: Add `icon-text` to [classes.js](https://github.com/sweetalert2/sweetalert2/blob/master/src/utils/classes.js#L32)

and then

`class="swal2-icon-text"` -> `class="${swalClasses['icon-text']}"`

**Comment by limonte** [comment]
Thanks a lot @Kuzcoo This is shipped in the latest version.

You helped us to close the oldest unresolved issue: https://github.com/sweetalert2/sweetalert2/issues/25

:tada: :tada: :tada:

**Comment by Kuzcoo** [comment]
@limonte : I'm glad I could help 😊

**Pull request merged**